src/Actions/Trackers/FocusTracker.cs

```diff
@@ -80,6 +84,48 @@ private void OnFocusChangedEventForSelectingElement(EventMessage message)
             }
         }
 
+        /// <summary>
+        /// microsoft/accessibility-insights-windows#1610: The "task switching"
+        /// interface that appears on Alt+Tab causes severe focus tracking
+        /// interference, especially on Sun Valley.
+        /// Therefore, this method detects whether this element is part of the
+        /// task switcher so that it can be filtered from focus tracking.
+        /// </summary>
+        /// <param name="element">The element to check.</param>
+        /// <returns>true if the element is part of the Alt+Tab task switching UI, false otherwise.</returns>
+        private bool IsTaskSwitcher(A11yElement element)
+        {
+            return (
+                _isWin11
+                && element != null
+                && element.ProcessName != "explorer"
+                && (
+                    DoesAncestryMatchCondition(
+                        element,
+                        "XamlExplorerHostIslandWindow", // "PANE" WINDOW THAT SOMETIMES TAKES FOCUS WHEN initiating Alt+Tab
+                        (DesktopElementAncestry anc) => anc.Items.Count == 1
+                    )
+                    || DoesAncestryMatchCondition(
+                        element,
+                        "ListViewItem", // Individual "task switching" item
+                        (DesktopElementAncestry anc) => anc.Items.Count > 0 && anc.Items[0].AutomationId == "SwitchItemListControl"
+                    )
+                )
+            );
+        }
+
+        private bool DoesAncestryMatchCondition(A11yElement element, string className, Func<DesktopElementAncestry, bool> f)
+        {
+            if (element.ClassName == className)
+            {
+                DesktopElementAncestry anc = new DesktopElementAncestry(Axe.Windows.Core.Enums.TreeViewMode.Control, element, true);
```

Comment:
I'd probably use `ancestry` instead of `anc` and `result` instead of `res`, but I won't choose that hill to die on